CMakeLists.txt

```diff
@@ -87,6 +87,30 @@ IF(WIN32)
 	)
 ENDIF(WIN32)
 
+# Get the current working branch
+IF(NOT($ENV{GITHUB_REF_NAME}))
```

Comment:
It's ok but you should understand what Steam version build from the different repo.

Reply:
I think, this is better than nothing, in this case we can ask you, what commit was this.